templates/sec_no_assignee_needinfo.txt

```diff
@@ -0,0 +1,2 @@
+There is no assignee and the security bug is opened for more than {{ extra['ndays'] }} {{ plural('day', extra['ndays']) }}.
```

Comment:
Maybe we could add a word (if we've something to say of course) about what to do when nobody can work on it.

Reply:
@marco-c what we could say here?

Reply:
We can suggest both, each one of them will be used in different scenarios.

BTW. We will not be posting reminders. So any of the suggested action will not have an impact on the same bug, it will serve as training for other bugs.

If you think the reminders are necessary, we could implement that by parsing the comment history to ensure enough gaps between reminders (easy to implement).